Lib/test/test_configparser.py

```diff
@@ -1555,6 +1555,23 @@ def test_source_as_bytes(self):
             "'[badbad'"
         )
 
+    def test_gh_107625(self):
+        #test key without value with extra whitespace within sections
+        lines = [
+            '[SECT]\n',
+            'KEY1\n',
+            ' KEY2 = VAL2\n', # note the Space before the key!
```

Comment:
Aren’t these two errors, that should be tested with two unit tests?

Reply:
A test for both read and read_file?

Reply:
No, one for solo `KEY1` (which is invalid if I understand correctly) and one for ` KEY2 = VAL2`

Reply:
I see from the bug report that both lines are necessary to cause the bug, ignore this!